### DISCUSSION TITLE
Plugin support the monitoring of IDSDM?

### DISCUSSION THREAD

**Discussion by Arjunan96** [bb-Ricardo/check_redfish#60] [Q&A]
Hi Ricardo, 

I have been asked to look into how to monitor the Dell Hardware IDSDM and in particular its redundancy status, SD Card state and SEL(system event logs) logs. 

I am aware that there is an association between redfish and IDSDM. Therefore does your plugin support the monitoring of IDSDM? 

Thanks, 
Arj

**Comment by bb-Ricardo** [comment]
Hi,

So far there is no particular support for IDSDM.

I have checked my available systems and could not find a System with IDSDM installed/enabled.

If you provide a Mockup of a System with IDSDM it would be possible to add the the plugin.

Cheers
Ricardo
